Docs - Add missing framework docs (#8013)
PostHog/posthog.com: contents/docs/libraries/svelte.md

---
title: Svelte
icon: ../../images/docs/integrate/frameworks/svelte.svg
---

> PostHog makes it easy to get data about traffic and usage of your [Svelte](https://astro.build/) app. Integrating PostHog into your site enables analytics about user behavior, custom events capture, session recordings, feature flags, and more.

Reviewer: ```suggestion
PostHog makes it easy to get data about traffic and usage of your [Svelte](https://svelte.dev/) app. Integrating PostHog into your site enables analytics about user behavior, custom events capture, session recordings, feature flags, and more.
```

        capture_pageleave: false
      }
    )
  }
  return
};
```

## Server-side setup

Install `posthog-node` using your package manager:

```bash
yarn add posthog-js
# or
npm install --save posthog-js
```

Then, initialize the PostHog Node client where you'd like to use it on the server side. For example, in a [load function](https://kit.svelte.dev/docs/load#page-data):

```js file=routes/+page.server.js
import { PostHog } from 'posthog-node';

export async function load() {
  const posthog = new PostHog('<ph_project_api_key>', 
  { host: '<ph_instance_address>' });
  posthog.capture({
    distinctId: 'distinct_id_of_the_user',
    event: 'event_name',
  })
  await posthog.shutdownAsync()
}
```

> **Note:** Make sure to always call `posthog.shutdownAsync()` after capturing events from the server-side. PostHog queues events into larger batches, and this call forces all batched events to be flushed immediately.

## Next steps

For any technical questions for how to integrate specific PostHog features into Svelte (such as analytics, feature flags, A/B testing, surveys, etc.), have a look at our [JavaScript Web](/docs/libraries/js) and [Node]((/docs/libraries/node)) SDK docs.

> Alternatively, we've also written the below tutorials to help get you started:

Reviewer: ```suggestion
Alternatively, the following tutorials can help you get started:
```
